### PR TITLE
build: bump year

### PR DESCRIPTION
**Pull request by CaerusKaru**
Patch version of https://github.com/angular/angular/pull/27880

### CODE REVIEW

**Comment by CaerusKaru** [comment]
cc @jenniferfell

**Comment by jenniferfell** [comment]
> Facebook stripped this out from their headers in a previous commit ([facebook/react#13593](https://github.com/facebook/react/pull/13593)). Wouldn't this be a bit easier if this was followed here?

Thanks, but we're going to continue using what's already been approved by Google's legal team.

**Review by jenniferfell — APPROVED** [review]
LGTM. @IgorMinar approved https://github.com/angular/angular/pull/27880 which is exactly the same changes just with a merge target that didn't include patch (stable).

**Comment by jenniferfell** [comment]
@petebacondarwin or @gkalpak : Can you approve as docs-infra? Thanks!

File: packages/router/README.md [hunk 1]
```diff
@@ -5,21 +5,5 @@ Managing state transitions is one of the hardest parts of building applications.
...
-# keep @angular/router fresh
-$ ./scripts/karma.sh
-
-# keep @angular/core fresh
-$ ../../../node_modules/.bin/tsc -p modules --emitDecoratorMetadata -w
-
-# start karma
-$ ./scripts/karma.sh
-```
+Read the dev guide [here](https://angular.io/guide/router).
```
Comment (petebacondarwin) [inline packages/router/README.md]: The changes to this file seem unrelated to this commit @jenniferfell. Is this intentional or an accident during cherry-picking?

**Review by petebacondarwin — CHANGES_REQUESTED** [review]

File: aio/src/app/layout/footer/footer.component.html [hunk 2]
```diff
@@ -1,23 +1,20 @@
...
-    Code licensed under an <a href="license" title="License text" >MIT-style License</a>.
-    Documentation licensed under
-    <a href="http://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a>.
-  </p>
-  <p>
-    Version {{versionInfo?.full}}.
-  </p>
-  <!-- TODO: twitter widget (but only on pages that use twitter) -->
+<p>
+  Super-powered by Google ©2010-2019.
```
Comment (gkalpak) [inline aio/src/app/layout/footer/footer.component.html]: 💪

File: packages/router/README.md (hunk 1 above)
Comment (CaerusKaru) [reply packages/router/README.md]: If you look at the original PR, you’ll see it’s a conscious edit that Igor made. I’m not sure exactly why he did it, but it’s there

Comment (jenniferfell) [reply packages/router/README.md]: We talked about this PR at eng standup today. Igor does remember approving the original PR, and he just missed the PR targets, so I think sticking with what's been approved is the fastest path. :-)

**Review by petebacondarwin — APPROVED** [review]

**Review by jasonaden — COMMENTED** [review]
For Docs team.

**Review by IgorMinar — APPROVED** [review]

**Comment by IgorMinar** [comment]
merge-assistance: this is a patch branch PR - no g3 impact. also pull approve seems stale. please rebase if necessary so that this PR lands.

**Review by alexeagle — APPROVED** [review]

**Comment by jenniferfell** [comment]
@CaerusKaru @IgorMinar Heads up: As you probably recall, this PR was specifically to take changes that had already been approved and merged into master, and also apply those same changes to patch (aka stable). Any additional changes made in this PR mean that you'll need to merge into master & patch.

**Pull request closed without merging**